cli/js/lib.deno.shared_globals.d.ts

```diff
@@ -1337,7 +1337,34 @@ declare class Worker extends EventTarget {
   terminate(): void;
 }
 
-declare namespace performance {
+declare type PerformanceEntryList = PerformanceEntry[];
+
+declare class Performance {
```

Comment:
Question: should these definitions be in `lib.deno.shared_globals.d.ts` or `lib.deno.ns.d.ts`? Won't it cause the same problem as in #6588?

Reply:
Yes, we hadn't established that before this, but you are right.  I will amend.

Reply:
Actually, it is a bit more subtle... because `dom` contains the APIs, only the delta between what is declared in `dom` needs to be accounted for.  So the latest commit does that, but in this case it is just duplicated and because of declaration merging, it normally is a redundant declaration, but would allow the User Timings Level 3 to be included.  I have added comments to `deno.ns` to make that clear why they exist there.